### ISSUE TITLE
What about using fast shutdown for fencing?

### ISSUE THREAD

**Issue by gbartolini** [cloudnative-pg/cloudnative-pg#3049]
Currently fencing follows the same logic of the shutdown, and issues a smart shutdown followed by a fast one.

However, fencing should be more assimilated to switchover, with a fast shutdown of up to `switchoverDelay` followed by immediate termination.

Comments?

**Comment by phisco** [comment]
Makes sense to me, I guess it would help making volumesnapshot based backups faster, is that what you are aiming for?

**Comment by gbartolini** [comment]
Yes, that too.
